skabase/SKALogger/SKALogger.py

```diff
@@ -11,17 +11,15 @@
 A generic base device for Logging for SKA. It enables to view on-line logs through the TANGO Logging Services
 and to store logs using Python logging. It configures the log levels of remote logging for selected devices.
 """
-# tango imports
-import tango
-from tango import DebugIt, DeviceProxy
-from tango.server import run, DeviceMeta, command
-
-# Additional import
 # PROTECTED REGION ID(SKALogger.additionnal_import) ENABLED START #
-from future.utils import with_metaclass
 # standard imports
 import os
 import sys
+from future.utils import with_metaclass
+# tango imports
```

Comment:
In other files there's a blank line before this comment - please keep it consistent.

Reply:
Checked and fixed in all other files.